VMT/VMT.md

```diff
@@ -0,0 +1,225 @@
+No stakeholder is supposed to deploy public patches before disclosure date.
+
+Once the email is sent, the KCSA Issue status should be set to *Fix committed*.
+At that point we can also add downstream stakeholders to the GitHub Issue, if they have an appropriate GitHub account.
+
+For non-embargoed, public vulnerabilities no separate downstream advance notification is sent.
+Instead, the KCSA Issue is set to *Fix committed* status once the CVE assignment is received and KCSA drafting begins immediately.
+
+## Open bug, Push patch
+
+> **FIXME** - stable maintainer depends on us defining the stable release support flows.
+
+In preparation for this, make sure you have a core developer and a stable maintainer available to help pushing the fix at disclosure time.
+
+On the disclosure hour, open bug, push patches to GitHub PR for review on master and supported stable branches, fast-track approvals (referencing the bug).
+
+Embargo reminder can be removed at that point.
+
+> **FIXME** - I suggest we formalise and set up a single place on our GitHub where we collect and advertise our known and previously fixed vulnerabilities - much like the [OpenStack one](https://security.openstack.org/ossalist.html). Then we can update the URL in the following paragraph.
+
+[MITRE's CVE Request form](https://cveform.mitre.org/) should be used again at this point, but instead select a *request type* of `Notify CVE about a publication` and fill in the coordinator's *e-mail address*, provide a *link to the advisory* (Use the links from the **FIXME** [Kata Containers Security Advisories page](https://github.com/kata-containers/community) if this is a formal KCSA), the *CVE IDs* covered, and the *date published*.
+Once more, fill in the *security code* at the bottom of the page and *submit request*.
+
+## Publish KCSA
+
+Shortly after pushing the patches (potentially waiting for the first test runs to complete), publish the advisory to the [Kata Containers Dev ML](mailto:kata-dev@lists.katacontainers.io) and slack channels.
+Wait until all patches are merged to supported branches before setting the KCSA Issue status to *Fix released*.
+
+## All patches merged
+
+Patches approved in code review do not necessarily merge immediately, but should be tracked closely until they do (if the Issue number is correctly identified in commit messages then it will be automatically updated to reflect this as well).
+Subsequent security point releases of affected software may then be tagged if warranted.
+
+# Incident Report Taxonomy
+
+The VMT is now using this classification list in order to assist vulnerability report triage, especially whenever a bug does not warrant an advisory.
+
+| Classes  | Outcome   | Description                               |
+| -------- | --------- | ----------------------------------------- |
+| Class A  | KCSA      | A vulnerability to be fixed in master and  all supported releases |
```

Comment:
If a user reports a major issue that only affects a single architecture, presumably we'd still release a fix for **all** architectures? Should this be noted somewhere?

Reply:
If it only affects one architecture, then I'm not sure if we will release for all architectures or not. Only if we had a rule that we must keep all architecture versions in sync maybe?
If we are only fixing one architecture, one could argue that for the sake of stability we *should not* apply the fix to the other architectures for instance.